docs/Interoperability_tests_guide/Configuring_test_cases.md

```diff
@@ -0,0 +1,146 @@
+
+### Using the search
+
+To search the list of configured test cases start typing your entry into the search field. The matching items will appear in the list.
+
+!!! tip
+    Note that if you select a test case from the filtered list, and then erase your entry from the search field, the selection will be carried over to the complete list view. Similarly, if you use the select all checkbox in the full list view, and then filter the list using the search, the selection will be carried over to the filtered list view.
+
+##	Viewing test case configuration
+
+Read this section to learn how to view the configuration of an individual test case:
+
+1.	From the navigation menu, select **Protocol tests configuration**.
+2.	In the list, find the test case you want to view and click on its name.
+
+![Viewing test case configuration](images/image021.png "Viewing test case configuration")
+
+3.	In the action list, expand the action items by clicking the ˅ icon.
+4.	To expand or collapse the complete action list, use the **Expand all** and **Collapse all** buttons.
+5.	Optionally, you can use the **Edit test case** button to edit your test case or click the **trash bin** icon to delete it.
+
+## Adding new test cases
+
+Read this section to learn how to add a new test case.
+
+1.	From the navigation menu, select **Protocol tests configuration**.
+2.	Click the **Add a new test case** button in the top-right corner:
+3.	Configure your test case:
+![Adding a new test case](images/image023.png "Adding a new test case")
+    -	Enter your **Test case name** (this field is mandatory).
+    -	Enter your **Test case description** (this field is optional).
+    -	Select your **Reference device** (this field is optional). You can either:
+         -	type the exact device name in the **Reference device** search field and hit ‘Enter’,
+         -	click **Select reference device** and select your device from the pop-up list. If you cannot see your device in the list, start typing its name in the search field to get matching results.
+4.	In the **Action list**, specify your actions:
+    -	To add your first action item, choose its name from the drop-down list, or type its name in the **Specify action** field.
+         -	Within the action, fill in the mandatory attributes field.
+    -	To add another action item, use the **Add action button** and specify your next action.
+    -	To change the order of actions within the test case, drag and drop the action item you want to move by using the **drag** icon.
+    -	To copy an action item, click the **copy** icon (except for the **Loop** action).
+    -	To delete an action item, click the **trash bin** icon.
+5.	If your test case is ready and all the mandatory fields are filled, click **Add a new test case**.
+
+!!! note
+    To learn more about individual test actions, see the Test case action description chapter.
+
+
+## Editing test cases
+
+Read this section to learn how to edit a test case.
```

Comment:
We should note that change will drop the historical results of the test. 
Do you know that case?